src/components/Header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import { HeaderContainer, LeftIconButton } from './Header.style';
+
+type HeaderProps = {
+  title: string;
+  leftIcon?: React.ReactNode;
+  onIconClick?: () => void;
+};
+
+const Header = ({ title, leftIcon, onIconClick }: HeaderProps) => {
+  const handleClick = () => {
+    onIconClick && onIconClick();
+  };
```

Comment:
```suggestion
  const handleClick = () => {
    onIconClick?.();
  };
```

1. optional 체이닝을 통해서 간결하게 표현해볼 수 있어요.
2. null 체크를 할 땐 falsy / truthy 를 사용하기보단 명시적으로 체크해보시는 걸 추천드려요. `onIconClick != null`

Reply:
[6b0ac8e](https://github.com/next-step/react-payments/pull/45/commits/6b0ac8ef98dd423cb97e49268400bb28853b2b23)